tests/_test_mscolab/test_utils.py

```diff
@@ -31,7 +31,9 @@
 from mslib.mscolab.conf import mscolab_settings
 from mslib.mscolab.models import Operation, MessageType
 from mslib.mscolab.seed import add_user, get_user
-from mslib.mscolab.utils import get_recent_op_id, get_session_id, get_message_dict, create_files, os_fs_create_dir
+from mslib.mscolab.utils import (get_recent_op_id, get_session_id,
+                                 get_message_dict, create_files,
+                                 os_fs_create_dir)
 
 
 class Message:
```

Comment:
I would like to get rid of this entire Message class. It is a very weird usage of a class (to kind of emulate an object of the actual Message class) and is not needed. The one instance where it is used could be replaced by an actual object of the actual class.

Reply:
This is unrelated to the topic of this PR, I opened #2353 for this.